locales/en/cos-ui.json

```diff
@@ -108,11 +111,15 @@
   "deleteInstance": "Delete instance",
   "deleteTypeNameMessage": "Type <1>{{connectorName}}</1> to confirm the deletion.",
   "deleting": "Deleting",
+  "deployment": "Deployment",
   "deploymentNamespace": "Deployment Namespace",
+  "deploymentStepDetail": "Select a Connectors namespace to host your Connectors instance. OpenShift options for creating a namespace are a Red Hat OpenShift Dedicated (OSD) trial cluster or a preview namespace in an OpenShift cluster managed by Red Hat.",
+  "deploymentStepDescription": "Alternatively, a Red Hat OpenShift Service on AWS (ROSA) cluster is an option that is  available with a subscription. <2>Talk to a Red</2>",
```

Comment:
Typo here at the end of the sentence, should it say something like "Talk to a Red Hat expert"?

Reply:
yeap missed in coping it should be "Talk to a Red Hatter to get a subscription", will update

Reply:
Updated.